plugins/woocommerce-blocks/assets/js/blocks/attribute-filter/test/block.tsx

```diff
@@ -109,11 +109,11 @@ const setup = ( params: SetupParams ) => {
 	const utils = render( <AttributeFilterBlock attributes={ attributes } />, {
 		legacyRoot: true,
 	} );
-	// We need to switch to React 17 rendering to allow these tests to keep passing, but as a result the React
-	// rendering error will be shown.
-	expect( console ).toHaveErroredWith(
+
+	expect( console ).not.toHaveErroredWith(
```

Comment:
Just mentioning, you should remove these lines, not add a `.not` to them. This was done because in our jest tests we use a WP utility which fails tests if a console error happens and the only way around it is to say we expect a specific console error. Without these lines the tests will still fail if there is a console error so its safe to remove them.

Reply:
You're right, @samueljseay. I just changed the lines from `.toHaveErroredWith()` to `.not.toHaveErroredWith()` temporary for testing purposes.